misc/userscripts/qute-pass

```diff
@@ -241,7 +292,7 @@ def main(arguments):
             break
     else:
         if not candidates:
-            stderr('No pass candidates for URL {!r} found! (I tried {!r})'.format(arguments.url, attempted_targets))
+            stderr('No pass candidates for URL {repr(arguments.url)} found! (I tried {repr(attempted_targets)})')
```

Comment:
I think you are missing an f on this one now?

Reply:
Good catch! The problem with f-strings is that they're equallid valid code as normal strings. Fixed in the latest commit.